src/getexchange.c

```diff
@@ -333,7 +332,15 @@ int getexchange(void) {
 		CONTEST_IS(CQWW) ||
 		CONTEST_IS(STEWPERRY)) {
 
-	    checkexchange(comment);
+	    checkexchange(comment, true);
+
+	    if (call_update && strlen(callupdate) >= 3) {
+		strcpy(hiscall, callupdate);
+		mvprintw(12, 29, "%s", spaces(MAX_CALL_LENGTH));
```

Comment:
How about using 'printcall()' for these two lines?

Reply:
Good point. I see know that call update interferes with autosend.

Reply:
Added to b6ec0f8